growthviz/charts.py

```diff
@@ -633,6 +670,15 @@ def bmi_with_percentiles(merged_df, bmi_percentiles, subjid):
 def param_with_percentiles(merged_df, subjid, param, wt_df, ht_df, bmi_df):
     """
     A version of bmi_with_percentiles() that provides the option of looking at wt or ht as well
+    
+    Parameters:
+    merged_df: (DataFrame) with subjid, bmi, include_height, include_weight, rounded_age
+               and sex columns
+    subjids: An list of the ids of the individuals to be plotted
+    param: (String) Whether to plot heights or weights. Expected values are "HEIGHTCM" or "WEIGHTKG"
```

Comment:
We should both check is it better to use an explicit Python type name literal? E.g., instead of this:

```python
param: (String) ...
```

...use the explicit "`str`", like this:

```python
param: (str) ...
```

Let's double check examples of this and pick one or the other.